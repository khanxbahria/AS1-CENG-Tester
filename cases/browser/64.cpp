#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("c", "c","w");
Tab B = Tab("ooo", "qq","y");
Tab C = Tab("b", "yy","ww");
Tab D = Tab("g", "oo","lll");
Tab E = Tab("zz", "bb","x");
Tab F = Tab("rr", "f","x");
Tab G = Tab("jjj", "ddd","l");
Tab H = Tab("p", "m","zzz");
Tab I = Tab("rrr", "n","bbb");
Tab J = Tab("hhh", "yy","xxx");
Tab K = Tab("v", "vv","u");
Tab L = Tab("iii", "g","p");
Tab M = Tab("cc", "ii","kkk");
Tab N = Tab("jj", "ddd","vvv");
Tab O = Tab("a", "uuu","g");
Tab P = Tab("ff", "zz","a");
Tab Q = Tab("qqq", "ll","cc");
Tab R = Tab("ss", "qqq","vvv");
Tab S = Tab("kkk", "vvv","m");
Tab T = Tab("iii", "ddd","bbb");
Tab U = Tab("ddd", "qqq","h");
Tab V = Tab("u", "rr","g");
Tab W = Tab("f", "vvv","mm");
Tab X = Tab("zzz", "w","e");
Tab Y = Tab("bbb", "vv","v");
Tab Z = Tab("eee", "m","lll");
a.newTab(A);
b.changeActiveTabTo(5);
c.newTab(B);
d.moveActiveTabTo(5);
a.newTab(C);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.newTab(D);
a.newTab(E);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.newTab(F);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(4);
b.newTab(G);
c.newTab(H);
d.newTab(I);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.newTab(J);
d.moveActiveTabTo(5);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
a.newTab(K);
b.moveActiveTabTo(5);
c.newTab(L);
d.newTab(M);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.newTab(N);
a.changeActiveTabTo(5);
b.newTab(O);
c.changeActiveTabTo(5);
d.newTab(P);
a.moveActiveTabTo(3);
b.newTab(Q);
c.newTab(R);
d.newTab(S);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.newTab(T);
a.newTab(U);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.moveActiveTabTo(3);
b.newTab(V);
c.moveActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.moveActiveTabTo(5);
b.newTab(W);
c.moveActiveTabTo(5);
d.newTab(X);
a.newTab(Y);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.newTab(Z);
d.changeActiveTabTo(3);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
d.changeActiveTabTo(5);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(5);
c.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
a.moveActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(4);
b.moveActiveTabTo(4);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(4);
c.changeActiveTabTo(4);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
d.moveActiveTabTo(0);
b.changeActiveTabTo(2);
c.changeActiveTabTo(2);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
d.changeActiveTabTo(2);
c.moveActiveTabTo(4);
a.moveActiveTabTo(0);
c.changeActiveTabTo(0);
b.moveActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.changeActiveTabTo(5);
b.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
a.changeActiveTabTo(2);
b.changeActiveTabTo(2);
c.moveActiveTabTo(2);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(3);
c.changeActiveTabTo(3);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
a.moveActiveTabTo(1);
b.moveActiveTabTo(1);
a.changeActiveTabTo(1);
c.moveActiveTabTo(1);
a.changeActiveTabTo(1);
c.moveActiveTabTo(1);
d.changeActiveTabTo(1);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
b.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(1);
c.moveActiveTabTo(1);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->getWindow(3);
brows->mergeAllWindows();
brows->closeWindow();
brows->newWindow();
brows->closeWindow();
brows->switchToWindow(8);
brows->closeEmptyWindows();
brows->newWindow();
brows->getWindow(1);
brows->newWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->switchToWindow(1);
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeWindow();
brows->getWindow(0);
brows->closeWindow();
brows->getWindow(0);
brows->newWindow();
brows->print();
brows->closeEmptyWindows();
brows->switchToWindow(5);
brows->closeWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->closeWindow();
brows->closeWindow();
brows->newWindow();
brows->print();
brows->newWindow();
brows->closeEmptyWindows();
brows->getWindow(5);
brows->newWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->newWindow();
brows->newWindow();
brows->getWindow(6);
brows->print();
brows->closeWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->switchToWindow(6);
brows->print();
brows->getWindow(5);
brows->getWindow(0);
brows->closeWindow();
brows->mergeAllWindows();
brows->switchToWindow(3);
brows->getWindow(4);
brows->closeWindow();
brows->newWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->closeWindow();
brows->switchToWindow(6);
brows->closeWindow();
brows->mergeAllWindows();
brows->switchToWindow(8);
brows->closeWindow();
brows->mergeAllWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->getWindow(7);
brows->print();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->getWindow(7);
brows->print();
brows->print();
brows->getWindow(5);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->switchToWindow(0);
brows->closeWindow();
brows->switchToWindow(1);
brows->getWindow(4);
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->getWindow(6);
brows->getWindow(8);
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->getWindow(1);
brows->newWindow();
brows->closeEmptyWindows();
brows->switchToWindow(9);
brows->closeWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->getWindow(3);
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeWindow();
brows->print();
brows->closeWindow();
delete brows;
}
	return 0;
}