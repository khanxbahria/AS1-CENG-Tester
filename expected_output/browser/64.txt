The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 3
















Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 3
















Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 3
















The browser is empty
The browser is empty
The browser is empty
The browser is empty
The browser is empty
